frontend/src/metabase/hooks/use-user-acknowledgement.ts

```diff
@@ -0,0 +1,29 @@
+import { useCallback } from "react";
+import { useUserKeyValue } from "./use-user-key-value";
+
+type UseUserAcknowledgementResult = [boolean, () => void, () => void];
+
+export const useUserAcknowledgement = (
```

Comment:
The name "userAcknowledgement" is a bit unclear. Maybe a comment here would be handy. Someone might think it has to do with acknowledging the user